includes/jetpack-settings/class-jetpack-settings.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Jetpack Settings
+ * 
+ * @package Cata\SEO
+ * @since 0.1.0
+ */
+
+namespace Cata\SEO;
+
+/**
+ * Jetpack Settings
+ */
+class Jetpack_Settings {
+	/**
+	 * Construct
+	 */
+	public function __construct() {
+		/**
+		 * Open Graph
+		 */
+		add_filter( 'jetpack_enable_open_graph', '__return_true' );
+
+		/**
+		 * Meta
+		 */
+		add_filter( 'jetpack_seo_meta_tags', array( __CLASS__, 'truncate_meta_description' ), 10, 1 );
+
+		/**
+		 * Sitemaps
+		 */
+		add_filter( 'jetpack_sitemap_image_skip_post', array( __CLASS__, 'jetpack_sitemap_image_skip_orphans' ), 10, 2 );
+		add_filter( 'jetpack_sitemap_image_skip_post', array( __CLASS__, 'jetpack_sitemap_image_skip_reusable_blocks' ), 12, 2 );
+
+		/**
+		 * Remove Concatenated Jetpack CSS
+		 */
+		add_action( 'wp_print_styles', array( __CLASS__, 'remove_jetpack_css' ), 1 );
+		add_action( 'wp_print_footer_scripts', array( __CLASS__, 'remove_jetpack_css' ), 1 );
+	}
+
+	/**
+	 * Truncate Meta Description
+	 * 
+	 * @link https://github.com/Automattic/jetpack/blob/8.3/modules/seo-tools/jetpack-seo.php#L197
+	 * @link https://stackoverflow.com/questions/79960/how-to-truncate-a-string-in-php-to-the-word-closest-to-a-certain-number-of-chara
+	 * 
+	 * @param array $meta - the meta tags Jetpack would like to output.
+	 * 
+	 * @return array $meta - possibly updated $meta with truncated description.
+	 */
+	public static function truncate_meta_description( array $meta ): array {
+		if ( ! isset( $meta['description'] ) || ! is_string( $meta['description'] ) ) {
+			return $meta;
+		}
+
+		if ( 150 >= strlen( $meta['description'] ) ) {
+			return $meta;
+		}
+
+		// Wrap to new lines at 150 characters.
+		$wrapped  = wordwrap( $meta['description'], 150, "\n" );
+		$position = strpos( $wrapped, "\n" );
+
+		if ( false === $position ) {
+			return $meta;
+		}
+
+		// Find first line.
+		$proposed = substr( $meta['description'], 0, $position );
+
+		if ( empty( $proposed ) ) {
+			return $meta;
+		}
+
+		return array_merge(
+			$meta,
+			array(
+				'description' => $proposed,
+			)
+		);
+	}
+	
+	/**
+	 * Remove Jetpack CSS
+	 */
+	public static function remove_jetpack_css(): void {
```

Comment:
I think this one isn't directly related to SEO and could be tricky to remember if we turn on Jetpack blocks on a site in the future.

Reply:
Good point, I removed this